test/cli_restart_test.go

```diff
@@ -46,16 +47,46 @@ func (suite *PouchRestartSuite) TestPouchRestart(c *check.C) {
 	}
 }
 
+// TestPouchRestartCreatedContainer is to verify the correctness of restarting a created container.
+// Pouch should be compatible with moby's API. Restarting a created container is allowed.
+func (suite *PouchRestartSuite) TestPouchRestartCreatedContainer(c *check.C) {
+	name := "TestPouchRestartCreatedContainer"
+
+	res := command.PouchRun("create", "--name", name, busyboxImage, "top")
+	defer DelContainerForceMultyTime(c, name)
+
+	res.Assert(c, icmd.Success)
+
+	command.PouchRun("restart", "-t", "1", name).Assert(c, icmd.Success)
+}
+
+// TestPouchRestartExitedContainer is to verify the correctness of restarting an exited container.
+// Pouch should be compatible with moby's API. Restarting an exited container is allowed.
+func (suite *PouchRestartSuite) TestPouchRestartExitedContainer(c *check.C) {
+	name := "TestPouchRestartExitedContainer"
+
+	//
+	res := command.PouchRun("run", "--name", name, busyboxImage, "sleep", "0.01")
+	defer DelContainerForceMultyTime(c, name)
+
+	res.Assert(c, icmd.Success)
+	time.Sleep(1 * time.Second)
```

Comment:
why this `sleep` is required? As you don't use `-d` in line 69, container is expected to be in `exited` state as soon as it returns, right?

Reply:
My fault, I added `-d`.